### DISCUSSION TITLE
Remove the "u/" prefix from comments

### DISCUSSION THREAD

**Discussion by anyone00** [Docile-Alligator/Infinity-For-Reddit#981] [Ideas]
Please provide an options to remove the **u/** prefix in comments, it is redundant. Maybe make that the default?

**Comment by severtheskyline** [comment]
Is there an option to deactivate it? If it isn't, I would prefer if that gets added before removing the u/ entirely.

**Comment by anyone00** [comment]
afaik there are no options currently.

**Comment by emvaized** [comment]
I think it was overlooked at some point, as Infinity provides an option to hide user prefix both in feed and on post details page. I would love to see that option for comments as well.
